### PR TITLE
1443-strip-progress-elements-from-the-metapanel

### PR DESCRIPTION
**Pull request by jberends**
Fixes #<issue>

## Checklist:
- [x] My code follows the pykechain style
    - [x] I added type hinting to all functions
    - [x] I added documentation for all public functions
    - [x] I locally used `tox` to check for dists and docs errors
    - [x] My code passes the `pep8` and `flake8` linting checks and no warnings
    - [x] I removed unused imports, using `Code > Optimize Imports` in PyCharm
    - [x] I used [`black`](https://github.com/psf/black) to format the new code
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] I committed fresh test cassettes
    - [x] I have proper test coverage (don't decline the coverage of the test)
- [x] I asked another teammate to review the code
- [x] I update the Changelog.md with the appropriate changes.

### CODE REVIEW

File: pykechain/client.py [hunk 1]
```diff
@@ -2572,6 +2579,12 @@ def create_widget(
         :raises IllegalArgumentError: when an illegal argument is send.
         :raises APIError: when an API Error occurs.
         """
+        if widget_type == WidgetTypes.PROGRESS:
```
Comment (raduiordache) [inline pykechain/client.py:2582]: This calls `validate_widget` which already has the same check on the widget_type

File: pykechain/client.py [hunk 2]
```diff
@@ -2461,6 +2463,11 @@ def _validate_widget(
         **kwargs,
     ) -> Dict:
         """Validate the format and content of the configuration of a widget."""
+        if widget_type == WidgetTypes.PROGRESS:
+            warnings.warn(
+                "The progress widget is not available in KE-chain from June 2024 onwards.",
+                DeprecationWarning,
+            )
```
Comment (raduiordache) [inline pykechain/client.py:2470]: i reckon we should just return inside this `if`, there is no point continuing validation

Comment (jberends) [reply pykechain/client.py:2470]: we need to return something according to the spec and we cannot return what is specced. So we maintain the warning (as discussed)

File: pykechain/client.py (hunk 1 above)
Comment (jberends) [reply pykechain/client.py:2582]: okay - agreed

**Review by jberends — COMMENTED** [review]
agreed with comments

**Pull request merged**